server/routes/user.js

```diff
@@ -52,11 +54,26 @@ router.post(
         .status(400)
         .send({ error: { msg: "User already exist", code: "400" } });
 
+    const transaction = await db.transaction();
     const newUser = await UserModel.create({
       name,
       email,
       password,
+      transaction,
     });
+    await UserListModel.create({
+      user_id: newUser.id,
+      title: "Shopping",
+      imageUrl: "https://image.flaticon.com/icons/png/512/1600/1600225.png",
+      transaction,
+    });
+    await UserListModel.create({
+      user_id: newUser.id,
+      title: "Wishlist",
+      imageUrl: "https://image.flaticon.com/icons/png/512/1600/1600225.png",
+      transaction,
+    });
```

Comment:
actually in this case I think it's okay if we don't link creating the default lists to user creation. If making the new lists fails we wouldn't want a user to be unable to sign up

Reply:
Alright. I'll make it a separate transaction tomorrow. It's late for me now so I'm going to head off to bed.